frontend/app/chat/components/ChatsList/components/ChatsListItem/components/UserButton.tsx

```diff
@@ -1,16 +1,25 @@
+import Image from "next/image";
 import Link from "next/link";
-import { MdPerson } from "react-icons/md";
 
 import { useSupabase } from "@/lib/context/SupabaseProvider";
 
+import { useGravatar } from "../hooks/useGravatar";
 import { sidebarLinkStyle } from "../styles/SidebarLinkStyle";
 
 export const UserButton = (): JSX.Element => {
   const { session } = useSupabase();
+  const { gravatarUrl } = useGravatar();
 
   return (
     <Link aria-label="account" className={sidebarLinkStyle} href={"/user"}>
-      <MdPerson className="text-4xl" />
+      <div className="relative w-8 h-8">
+        <Image
```

Comment:
So what happens when it doesnt exist on gravatar ?

Reply:
It displays a default image representing an anonymous user :
<img width="224" alt="image" src="https://github.com/StanGirard/quivr/assets/67386567/62faffbb-8545-4020-ab05-6a30d32ec4f8">